compiler_opt/es/policy_utils_test.py

```diff
@@ -220,6 +220,20 @@ def test_tfpolicy_and_loaded_policy_produce_same_variable_order(self):
     # assert that they result in the same order of values
     np.testing.assert_array_almost_equal(tf_params, loaded_params)
 
+  def test_convert_to_tflite(self):
+    policy_save_path, _, _ = self._save_inlining_policy()
+    saved_model_path = os.path.join(policy_save_path, self.POLICY_NAME)
+
+    output_bytes = self.params.tobytes()
+
+    scratch_dir = self.create_tempdir()
+    tflite_dir = policy_utils.convert_to_tflite(output_bytes, scratch_dir,
+                                                saved_model_path)
+
+    self.assertTrue(os.path.exists(os.path.join(tflite_dir, 'model.tflite')))
+    self.assertTrue(
+        os.path.exists(os.path.join(tflite_dir, 'output_spec.json')))
```

Comment:
can you test the resulting policy's weights match what you started with?

Reply:
Not easily. We only have support for loading the parameters from a saved model, not from a TFLite conversion of that saved model.

Reply:
if you split `convert_to_tflite` at line 150, would it work to test the bit above line 150? The stuff below is fairly standard.

Reply:
I'm not sure it makes too much sense to split. I've added an assertion that the saved model produced during the process has the correct parameters which should have the equivalent effect.